File-Formats/PDF/Convert-HTML-To-PDF/aspnet-mvc.md

```diff
@@ -13,54 +13,59 @@ The Syncfusion HTML to PDF converter is a .NET library used to convert HTML or w
 
 ## Steps to convert HTML to PDF document in ASP.NET MVC
 
-Create a new C# ASP.NET Web Application (.NET Framework) project.
+1. Create a new C# ASP.NET Web Application (.NET Framework) project.
 ![convert_HtmltoPdf_ASP.NET_MVC1](htmlconversion_images/aspnetmvc1.png)
 
-Install Syncfusion.HtmlToPdfConverter.AspNet.Mvc5  NuGet package as reference to your .NET applications from NuGet.org.
+2. Install [Syncfusion.HtmlToPdfConverter.AspNet.Mvc5](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.AspNet.Mvc5)  NuGet package as reference to your .NET applications from [NuGet.org](https://www.nuget.org/).
 ![convert_HtmltoPdf_ASP.NET_MVC2](htmlconversion_images/aspnetmvc2.png)
 
-Include the following namespaces in that HomeController.cs file.
+3. Include the following namespaces in that HomeController.cs file.
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-using Syncfusion.Pdf;
-using Syncfusion.HtmlConverter;
+   using Syncfusion.Pdf;
+   using Syncfusion.HtmlConverter;
+   using System.IO;
 
-{% endhighlight %}
+   {% endhighlight %}
 
-Add a new button in the Index.cshtml as shown below.
+4. Add a new button in the Index.cshtml as shown below.
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-@{Html.BeginForm("ExportToPDF", "Home", FormMethod.Post);
-{
-<div>
-    <input type="submit" value="Convert PDF" style="width:150px;height:27px" />
-</div>
-}
-Html.EndForm();
-}
+   @{Html.BeginForm("ExportToPDF", "Home", FormMethod.Post);
+       {
+          <div>
+              <input type="submit" value="Convert PDF" style="width:150px;height:27px" />
+          </div>
+       }
+       Html.EndForm();
+   }
 
-{% endhighlight %}
+   {% endhighlight %}
 
-Add a new action method ExportToPDF in HomeController.cs and include the below code snippet to convert HTML to PDF.
+5. Add a new action method ExportToPDF in HomeController.cs and include the below code snippet to convert HTML to PDF.
```

Comment:
Add a new action method named ExportToPDF in HomeController.cs and include the below code snippet to convert HTML to PDF document.

Reply:
content changed